DaemonRenderer/Source/Src/Windowing/Screen.cpp

```diff
@@ -70,12 +55,28 @@ Screen::Screen(GLFWmonitor* in_handle):
             modes[i].refreshRate
         });
     }
+
+    glfwSetMonitorUserPointer(in_handle, this);
+}
+
+Screen::Screen(Screen&& in_move) noexcept:
+    m_logger        {in_move.m_logger},
```

Comment:
Both the logger and the handle can be moved too here since they are pointers, the actual objects will remain untouched

Reply:
A pointer is trivially copyable so moving it is less optimized.